core/src/main/java/org/apache/cxf/service/model/EndpointInfo.java

```diff
@@ -94,7 +95,12 @@ public String getAddress() {
 
     public void setAddress(String addr) {
         if (null == address) {
-            address = EndpointReferenceUtils.getEndpointReference(addr);
+            // address = EndpointReferenceUtils.getEndpointReference(addr); loads jaxb and we want to avoid it
+            final EndpointReferenceType reference = new EndpointReferenceType();
+            final AttributedURIType a = new AttributedURIType();
+            a.setValue("/");
```

Comment:
This should be: a.setValue(address);

Reply:
Hmm, sure? address is null here so it defaults to "/" but good catch i forgot the else ;)